src/runtime/parallel/master.js

```diff
@@ -100,7 +106,7 @@ export default class Master {
   }
 
   run(numberOfSlaves, done) {
-    this.eventBroadcaster.emit('test-run-started')
+    this.eventBroadcaster.emit('test-run-started', { numberOfSlaves })
```

Comment:
Can we set startTime as the first line in this function instead of in parseSlaveMessage?

Also, thoughts on adding `parallel: true` to the test-run-started event? Then the serial runner can add `parallel: false` to its version and the summary formatter could just save testRunStartedData and would check if data.parallel then do the special printing.

Reply:
this sounds good to me, will make these updates